### PR TITLE
Removed the dependancy on length of the GuestName

### PR DESCRIPTION
**Pull request by sreeman-ambati-4421**

### CODE REVIEW

File: network-wrapper [hunk 1]
```diff
@@ -153,15 +157,15 @@ done < /proc/mounts
 }
 
 # Try to find a cgroup matching exactly the provided name.
-N=$(find "$CGROUPMNT" -name "$GUESTNAME" | wc -l)
+N=$(find "$CGROUPMNT" -name "$GUEST_LABEL_NAME" | wc -l)
```
Comment (abstractor84) [inline network-wrapper:160]: This needs to be tested in both centos and ubuntu, the cgroup will have container id and not name.

**Review by abstractor84 — CHANGES_REQUESTED** [review]
1. Need to be tested in centos 7.x and Ubuntu
2. cgroup naming in centos and ubuntu need to be verified, as they use container id (short) and not the label name

File: network-wrapper [hunk 2]
```diff
@@ -28,11 +28,15 @@ RUN_FILE=/var/run/network-wrapper
...
+# GUESTNAME can be Container Name, or Short UUID or Long UUID.
+STAT_GUEST=$(docker inspect ${GUESTNAME} >/dev/null 2>&1; echo $?)
 if [[ "$STAT_GUEST" == "0" ]]; then
-    GUEST_LABEL_NAME=$(docker ps -a  --format "{{.ID}}\t{{.Names}}" | grep ${GUESTNAME:0:12} | awk '{print $NF}')
+    GUEST_LABEL_NAME=$(docker inspect ${GUESTNAME} --format='{{ .Name }}')
+    # Will return Guest Name as follows: /HSM_1_centos
+    GUEST_LABEL_NAME=${GUEST_LABEL_NAME:1}
 else
-    GUEST_LABEL_NAME=${GUESTNAME}
+    # Should we really continue ?
```
[the remark this answers is not in the document]
Comment (sreeman-ambati-4421) [reply network-wrapper:38]: Addressed. Removed exit.

File: network-wrapper (hunk 1 above)
Comment (sreeman-ambati-4421) [reply network-wrapper:160]: Have not verified in Centos.

**Pull request merged**